### ISSUE TITLE
Properly display GIF profile pictures

### ISSUE THREAD

**Issue by kenokenobingo** [DagAgren/toot#385]
Animated GIF profile pictures are displayed as still images.

**Comment by VirtualWolf** [comment]
And it should remain that way IMO. :P Or at least have an option to turn it off, I can't stand random things flashing or moving around my UIs when I don't want them to.

**Comment by kenokenobingo** [comment]
@VirtualWolf, agreed! But an option would be nice (my profile pic just looks stupid now, haha). :)

**Comment by DagAgren** [comment]
This is actually really tricky, because iOS has no built-in support for GIF loading or playing, and the third-party implementations I've seen seem really wasteful and inefficient.

I have implemented my own GIF loader in the past, but it's a big task, and I am not sure it is worth the cost, unless Mastodon starts providing them as video files.
